enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -0,0 +1,434 @@
+---
+title: gateway-api-with-cluster-ingress-operator
+authors:
+  - Miciah
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2022-12-13
+last-updated: 2022-12-13
+tracking-link:
+  - https://issues.redhat.com/browse/NE-1105
```

Comment:
Isn't this also an EP for:
https://issues.redhat.com/browse/NE-1107
https://issues.redhat.com/browse/NE-1108
Or are you envisioning them being separate EPs?

Reply:
I intend for this EP to cover all three.  Maybe I should just link to the epic?

Reply:
It is a list.  I'll list all three (NE-1105, NE-1107, and NE-1108).